server/src/domain/media/media.service.spec.ts

```diff
@@ -259,6 +259,23 @@ describe(MediaService.name, () => {
       );
     });
 
+    it('should not scale resolution if no target resolution', async () => {
+      mediaMock.probe.mockResolvedValue(probeStub.videoStream2160p);
+      configMock.load.mockResolvedValue([
+        { key: SystemConfigKey.FFMPEG_TRANSCODE, value: 'all' },
+        { key: SystemConfigKey.FFMPEG_TARGET_RESOLUTION, value: '' },
```

Comment:
This would actually be the value `original`, right?

Reply:
Yup, the text value was originally just an empty string but I forgot to update the test when I changed it..

Reply:
Ah, makes sense. Looks good now!